<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>department</class>
 <widget class="QWidget" name="department">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>472</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="id">
         <property name="text">
          <string>id</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="firstName_2">
         <property name="text">
          <string>firstName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lastName_2">
         <property name="text">
          <string>lastName</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="phoneNumber_2">
         <property name="text">
          <string>Phone number</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QLineEdit" name="lineId"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineFirstName_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="lineLastName_2"/>
       </item>
       <item>
        <widget class="QLineEdit" name="linePhoneNumber_2"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="add_2">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="delete_3">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QTableView" name="tableView_2"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
